app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsFragment.kt

```diff
@@ -73,7 +75,15 @@ class TransactionDetailsFragment : BaseViewBindingFragment<FragmentTransactionDe
                 }
                 is BaseStateViewModel.ViewAction.ShowError -> {
                     binding.refresh.isRefreshing = false
-                    //TODO: handle error here
+
+                    binding.content.visibility = View.GONE
+
+                    snackbar(requireView(), viewAction.error.message ?: getString(R.string.error_cannot_load_tx_details))
+                    Timber.e(viewAction.error)
```

Comment:
we already log error in base state view model. In general, I think view model is a better place for error logs that are not intended for an end-user.